1.19_linux/runtime/sys_linux_amd64.s.patch

```diff
@@ -21,7 +21,7 @@ TEXT runtime·open_trampoline(SB),NOSPLIT,$0
 	MOVL	12(DI), DX		// arg 3 mode
 	MOVQ	0(DI), DI		// arg 1 pathname
 	XORL	AX, AX			// vararg: say "no float args"
-	CALL	c_open(SB)
```

Comment:
I'm afraid this might be a burden on you, but I think it'd be nice to have a separate PR to rename the existing `c_` functions to `hitsumabushi_` functions